soft4pes/model/grid/rl_grid.py

```diff
@@ -117,44 +116,9 @@ def get_grid_voltage(self, kTs):
         vg = abc_2_alpha_beta(vg_abc)
         return vg
 
-    def update_state(self, u, matrices, kTs):
-        """
-        Get the next state of the grid.
-
-        Parameters
-        ----------
-        u : 1 x 3 ndarray of floats
-            Converter three-phase switch position.
-        matrices : SimpleNamespace
-            A SimpleNamespace object containing matrices A, B1 and B2 of the 
-            state-space model.
-        kTs : float
-            Current time [s].
-
-        Returns
-        -------
-        1 x 2 ndarray of floats
-            Next state of the grid [p.u.].
-        """
-
+    def update_state(self, matrices, uk_abc, kTs):
         vg = self.get_grid_voltage(kTs)
-        self.save_data(vg, kTs)
-        x_kp1 = np.dot(matrices.A, self.x) + np.dot(matrices.B1, u) + np.dot(
-            matrices.B2, vg)
-        self.x = x_kp1
-
-    def save_data(self, vg, kTs):
-        """
-        Save system data.
-
-        Parameters
-        ----------
-        vg : 1 x 2 ndarray of floats
-            Grid voltage in alpha-beta frame [p.u.].
-        kTs : float
-            Current time [s].
-        """
-
-        self.sim_data['x'].append(self.x)
-        self.sim_data['vg'].append(vg)
-        self.sim_data['t'].append(kTs)
+        x_kp1 = np.dot(matrices.A, self.x) + np.dot(
```

Comment:
Do we have an option to have the grid voltage as part of the state? In such a case the state-space model would be x(k+1) =  A x(k) + B u(k)

The current option is fin. I am just wondering if we have also the second option available